bin/worker.php

```diff
@@ -32,23 +30,19 @@
 
 $a = new App(dirname(__DIR__));
 
-Config::load();
-
 $lang = L10n::getBrowserLanguage();
 L10n::loadTranslationTable($lang);
 
 // Check the database structure and possibly fixes it
 check_db(true);
 
 // Quit when in maintenance
-if (Config::get('system', 'maintenance', false, true)) {
+if (!$a->getMode()->has(App\Mode::MAINTENANCEDISABLED)) {
 	return;
 }
 
 $a->setBaseURL(Config::get('system', 'url'));
 
-Addon::loadHooks();
```

Comment:
AFAIK there had been issues when not doing this. I'm unsure if it had been from the daemon or cron call.

Reply:
It is always done in `App->reload()` during each `new App()` call, so this is superfluous now.